src/i18n/en/nav.ts

```diff
@@ -29,22 +29,22 @@ export default [
 	{ text: 'Static Assets', slug: 'guides/imports', key: 'guides/imports' },
 	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
 	{ text: 'Deploy', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'Server-side Rendering (SSR)', slug: 'guides/server-side-rendering', key: 'guides/server-side-rendering' },
 
 
 	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
 	{ text: 'Configuring Astro', slug: 'guides/configuring-astro', key: 'guides/configuring-astro' },
+	{ text: 'TypeScript', slug: 'guides/typescript', key: 'guides/typescript' },
+	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
+	{ text: 'UI Frameworks', slug: 'core-concepts/framework-components', key: 'core-concepts/framework-components' },
+	{ text: 'Sharing State', slug: 'core-concepts/sharing-state', key: 'core-concepts/sharing-state' },
 	{ text: 'CSS & Styling', slug: 'guides/styling', key: 'guides/styling' },
+	{ text: 'Images', slug: 'guides/images', key: 'guides/images' },
 	{ text: 'Fonts', slug: 'guides/fonts', key: 'guides/fonts' },
 	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
-	{ text: 'Environment Variables', slug: 'guides/environment-variables', key: 'guides/environment-variables' },
-	{ text: 'Images', slug: 'guides/images', key: 'guides/images' },
 	{ text: 'Import Aliases', slug: 'guides/aliases', key: 'guides/aliases' },
-	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
+	{ text: 'Environment Variables', slug: 'guides/environment-variables', key: 'guides/environment-variables' },
 	{ text: 'RSS', slug: 'guides/rss', key: 'guides/rss' },
```

Comment:
I like the idea of ordering these loosely by:
- How common it is to use a feature
- "order of operations": if one item is a prereq to use another, move it forward
I think Sharing State is less common than CSS, Images, and Fonts, so I'd move that down.
Tweaking TypeScript is less common of a goal than using UI Frameworks, so I'd move that down.
Configuring Astro goes deep into the weeds of config. It's more likely that you'd want to include a UI Framework or an integration than to configure something that _isn't_ covered by those pages. So I'd also move that down.

Here's _one_ possible version of incorporating those three changes:

```suggestion
	{ text: 'Integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
	{ text: 'UI Frameworks', slug: 'core-concepts/framework-components', key: 'core-concepts/framework-components' },
	{ text: 'Configuring Astro', slug: 'guides/configuring-astro', key: 'guides/configuring-astro' },
	{ text: 'TypeScript', slug: 'guides/typescript', key: 'guides/typescript' },
	{ text: 'CSS & Styling', slug: 'guides/styling', key: 'guides/styling' },
	{ text: 'Images', slug: 'guides/images', key: 'guides/images' },
	{ text: 'Fonts', slug: 'guides/fonts', key: 'guides/fonts' },
	{ text: 'Data Fetching', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
	{ text: 'Sharing State', slug: 'core-concepts/sharing-state', key: 'core-concepts/sharing-state' },
	{ text: 'RSS', slug: 'guides/rss', key: 'guides/rss' },
  { text: 'Import Aliases', slug: 'guides/aliases', key: 'guides/aliases' },
	{ text: 'Environment Variables', slug: 'guides/environment-variables', key: 'guides/environment-variables' },
```

I could also see moving "Configuring Astro" and "Typescript" down to form a little "config files" group:
- Configuring Astro
- TypeScript
- Import Aliases
- Environmental Variables

Reply:
I could get behind a little Config group. Would you even add its own heading? Part of me really kind of wants configuring Astro to be higher, because it seems like configuring might be a thing you'd want to do, but I think you're right. That's a bit more advanced and not what people are jumping to immediately. 

Let me try a reorder based on this and see what I come up with, and how closely it matches this.

Reply:
Pushed another option, including a new section... waiting for checks, but let's see how this looks @Jutanium

Reply:
It looks great! Little Config Group is all grown up with its own header